holoviews/element/util.py

```diff
@@ -25,17 +25,6 @@
     xr = None
 
 
-def toarray(v, index_value=False):
-    """
-    Interface helper function to turn dask Arrays into numpy arrays as
-    necessary. If index_value is True, a value is returned instead of
-    an array holding a single value.
-    """
-    if dask and isinstance(v, dask.array.Array):
-        arr =  v.compute()
-        return arr[()] if index_value else arr
-    else:
-        return v
```

Comment:
We now support dask arrays by proxy through xarray, so there's not really any need.